libs/utils/src/lib/language/nb.ts

```diff
@@ -156,15 +156,13 @@ export const nb = {
     searchFieldCodeList: 'Søkefelt kodeliste',
     enableFilter: 'Aktiver filter',
     description: {
-      general:
-        'Oversikt over beskrivelser av datasett, begrep, apier og informasjonmodeller. Innholdet blir levert av ulike virksomheter, offentlige og private.',
-      conceptCatalog:
-        'Oversikt over beskrivelser av datasett, begrep, apier og informasjonmodeller. Innholdet blir levert av ulike virksomheter, offentlige og private.',
+      general: 'Her kan du administrere innstillinger og innhold på tvers av katalogene.',
+      conceptCatalog: 'Her kan du administrere innstillinger og innhold for begrepskatalogen.',
     },
     manage: {
       internalFields: 'Her kan du legge til, endre og fjerne interne felt som er spesifikke for din virksomhet.',
-      editableFields: 'Administrer editerbare felt',
-      codeList: 'Administrer kodeliste',
+      editableFields: 'Her kan du velge kodelister for felt som publiseres eksternt, som fagområde.',
```

Comment:
Jeg vil tippe det neste vi legger til på den siden er muligheten til å deaktivere/aktivere visse felter. Er uansett lite sannsynlig at velging av kodelister for noen felt er det eneste vi legger til der, så føler den teksten kommer til å være misvisende senere, men ok atm :shrug:

Reply:
Fikk teksten fra @KjerstiSteien, men ja, kan evt. endre når resten av funksjonaliteten er på plass?